src/config.py

```diff
@@ -10,7 +10,7 @@ class Config:
     See: https://docs.platform.sh/development/variables.html
 
     The following are 'magic' properties that may exist on a Config object. Before accessing a property, check its
-    existence with hasattr(config, variableName). Attempting to access a nonexistent variable will throw an exception.
+    existence with config['variableName']. Attempting to access a nonexistent variable will throw an exception.
```

Comment:
Sorry, I was unclear about this. `__getitem__` should be for documented for internal use only, meaning instead of using the "private" method `self._get_value()`, you can just say `self[""]`. `hasattr()` is still correct for checking existence of an attribute because if I'm understanding this correctly `_get_value()`/`__getitem__` is intended for the **raw** values, while the "magic" attributes are intended for the cleaned up values that a user might use, correct?

Reply:
Yeah, that's right - just moving too fast on that one.